### DISCUSSION TITLE
[Fast Data][API Portal] Projection name appears wrongly formatted

### DISCUSSION THREAD

**Discussion by flower-of-the-bridges** [mia-platform/community#238] [Q&A] [answered]
### Which section?

Console

### Which is the version?

v10.6.0

### Description

If I have the following situation in my console configuration:

- system of record: `books`
- projection: `pr_reviews`

The API Portal shows the endpoint of a Fast Data Projection with the following name: `BooksPr_reviews`

### Actual Outcome

![image](https://user-images.githubusercontent.com/38134891/234605018-516e49de-317d-4eb3-86f7-be35f54b0064.png)

### Expected Outcome

The name of the endpoint should be:

```
books pr_reviews
```

**Comment by fedepini** [comment, marked as answer]
Hi, @giovf96!

Thanks for the catch, before suggesting any kind of solution to solve this bug, can you please tell us the repro steps and which version of the API Portal are you using? As a general advice, we suggest you to update it if it is an old version.

We talk about the API Portal because we believe the problem could be related to its front end, especially the way projection names are translated into readable labels. If so, Fast Data would not be involved since it only passes the endpoint name to the API Portal.

**Reply by flower-of-the-bridges** [reply]
Hi @fedepini 
The steps to reproduce the issue are the following:
From the console:
* create a projection in the fast data section
* create an endpoint in the API gateway section
* deploy the project and go to the API portal

The API portal version is `1.16.2`

**Reply by flower-of-the-bridges** [reply]
I also take a look at how the crud service generates the documentation tag, and it seems that this is the regex that generates the wrong tag:

https://github.com/mia-platform/crud-service/blob/da2d5ccf5ac053fb418b24e6744447f76d79632c/lib/JSONSchemaGenerator.js#L210
